scripts/gen_vimdoc.py

```diff
@@ -132,6 +134,7 @@
             # `shared` functions are exposed on the `vim` module.
             'shared': 'vim',
             'uri': 'vim',
+            'keymap': 'vim.keymap',
```

Comment:
do keymaps justify a new "namespace"? can these functions live on `vim` directly (even if we internally keep them in a `keymap.lua` module, don't need to expose users to that)?